src/content/en/fundamentals/security/credential-management/reference.md

```diff
@@ -0,0 +1,207 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2017-04-14 #}
+{# wf_published_on: 2017-04-14 #}
+
+# API Reference {: .page-title }
```

Comment:
Hey, Joe.

Another good question.

These tables are in the existing docs-- I just moved them out as they get in the way of the flow of the step-by-step narrative.

It is easy to link to MDN docs, and I can make sure the MDN docs have everything needed.

Gonna confirm with Eiji though-- 

@agektmr - do you want to keep the reference tables in the doc, or are you OK with linking out to MDN? I do find it helpful referencing them, so maybe I could just include the tables in the docs, rather than moving them to their own reference section?